librz/crypto/des.c

```diff
@@ -1,3 +1,8 @@
+// SPDX-FileCopyrightText: 2015 seu <seu@panopticon.re>
+// SPDX-FileCopyrightText: 2015 condret <condret@runas-racer.com>
+// SPDX-FileCopyrightText: 2017 Giovanni <wargio@libero.it>
```

Comment:
@wargio I think he wants to be mentioned as `wargio` or `deroad`. Please tell us.

Reply:
deroad svp